backend-project/small_eod/generic/tests.py

```diff
@@ -96,3 +101,59 @@ def test_factories_simple(self):
     def test_factories_many(self):
         for x in range(1, self.FACTORY_COUNT):
             self._test_factory_object(msg='many', count=x)
+
+
+    def test_print_to_console(self):
+        """
+        Show in the console how the data looks like.
+        Just making sure that frontend has the right data
+        to work with.
+        """
+        print(f"\n{model_to_dict(self.FACTORY())}")
```

Comment:
Nie wierzę, że ktoś to będzie czytać. Nie na tym polegają testy automatyczne, aby je ciągle czytać. Już lepiej zapisać wynik i porównywać czy nie zmienił się.

Reply:
Za pierwszym razem sie przydaje, moze usuniemy po tym jak juz dodam wszystkie factories?

Reply:
Gdybyś podał wszystkie parametry do factory i skopiował wynik model to dict, aby następnie porównać to mielibyśmy pewność, że struktura obiektu nie uległa zmianie.

Reply:
Wlasnie zalezy mi na sprawdzaniu tych automatycznych factory.

Reply:
Wlasnie pozwolilo mi to zobaczyc ze nalezy uzyc metody abstract a nie mixinu.
````python
test_print_to_console (small_eod.institutions.tests.InstitutionFactoryTestCase) ...
{'id': 275, 'created_by': None, 'modified_by': None, 'name': 'name-10', 'external_identifier': 570, 'administrative_unit': 10, 'address': 550}
````
````python
class TimestampUserFactoryMixin(factory.Factory):
    created_by = factory.SubFactory(UserFactory)
    modified_by = factory.SubFactory(UserFactory)

    class Meta:
        abstract = True
````
https://github.com/FactoryBoy/factory_boy/issues/165